ci/cloudbuild/cache.sh

```diff
@@ -111,8 +111,14 @@ function save_cache() {
     return 0
   fi
   io::log "Saving ( ${paths[*]} ) to ${PRIMARY_CACHE_URL}"
-  tar -czf - "${paths[@]}" | gsutil cp - "${PRIMARY_CACHE_URL}"
```

Comment:
I wish I'd thought of that earlier. That's a good point. I think I'm going to keep this change to let gsutil do retries, but I also changed the `cache.sh` steps in the `cloudbuild.yaml` file so that cache saving/restoring never breaks the build. These are nice-to-have steps only.

PTAL